### ISSUE TITLE
...TinaRemark fragment does not exist

### ISSUE THREAD

**Issue by amykapernick** [tinacms/tinacms#1127]
## Description

Getting an error with TinaCMS Fragment in Gatsby using `gatsby-tinacms-remark` v0.7.16
 (v`0.7.15`, `0.7.16-alpha.3` and `0.7.0-canary.0` works). Have tried switching versions and this only occus on v0.7.16

```
There was an error in your GraphQL query:

The fragment "TinaRemark" does not exist.
```

### Steps to reproduce

- Clone gatsby starter <https://github.com/tinacms/gatsby-starter-tinacms>
- Upgrade `gatsby-tinacms-remark` package to use v`0.7.16`

### Expected result

It should render the page

### Actual result

Displays an error

```
There was an error in your GraphQL query:

The fragment "TinaRemark" does not exist.

  131 |     site {
  132 |       siteMetadata {
  133 |         title
  134 |         author
  135 |       }
  136 |     }
  137 |     markdownRemark(fields: { slug: { eq: $slug } }) {
  138 |       id
  139 |       excerpt(pruneLength: 160)
  140 |       html
> 141 |       ...TinaRemark
      |       ^^^^^^^^^^^^^
  142 |       frontmatter {
  143 |         title
  144 |         date(formatString: "MMMM DD, YYYY")
  145 |         description
  146 |       }
  147 |     }
  148 |   }
  149 | `
  150 | 

File: /c/Users/amyka/web/gatsby-starter-tinacms/src/templates/blog-post.js
```

#### Tinacms Package Versions
```text
├─ @tinacms/alerts@0.1.3
├─ @tinacms/api-git@0.6.0
├─ @tinacms/core@0.8.0
├─ @tinacms/fields@0.10.2
├─ @tinacms/form-builder@0.3.3
│  ├─ @tinacms/forms@^0.4.2
│  ├─ @tinacms/react-core@^0.2.12
├─ @tinacms/forms@0.4.2
├─ @tinacms/git-client@0.5.0
├─ @tinacms/icons@0.10.0
├─ @tinacms/media@0.1.0
├─ @tinacms/react-alerts@0.1.4
├─ @tinacms/react-core@0.2.12
│  ├─ @tinacms/core@^0.8.0
│  └─ @tinacms/forms@^0.4.2
├─ @tinacms/react-forms@0.1.4
│  ├─ @tinacms/form-builder@^0.3.3
├─ @tinacms/react-modals@0.1.4
├─ @tinacms/react-screens@0.1.3
├─ @tinacms/react-sidebar@0.1.4
│  ├─ @tinacms/core@^0.8.0
│  ├─ @tinacms/forms@^0.4.2
├─ @tinacms/react-toolbar@0.1.2
│  ├─ @tinacms/core@^0.8.0
│  ├─ @tinacms/fields@^0.10.2
│  ├─ @tinacms/forms@^0.4.2
├─ @tinacms/styles@0.4.2
├─ gatsby-plugin-tinacms@0.1.36
│  └─ tinacms@^0.19.2
├─ gatsby-tinacms-git@0.3.0
│  ├─ @tinacms/api-git@^0.6.0
│  └─ @tinacms/git-client@^0.5.0
├─ gatsby-tinacms-json@0.8.10
├─ gatsby-tinacms-remark@0.7.16
├─ react-tinacms-editor@0.5.2
│  ├─ @tinacms/core@^0.8.0
│  ├─ @tinacms/fields@^0.10.2
│  ├─ @tinacms/icons@^0.10.0
│  ├─ @tinacms/react-forms@0.1.2-alpha.0
│  ├─ @tinacms/react-forms@0.1.2-alpha.0
│  │  ├─ @tinacms/form-builder@^0.3.2-alpha.0
│  ├─ @tinacms/styles@^0.4.2
├─ react-tinacms@0.8.1
│  ├─ @tinacms/core@^0.4.4
│  ├─ @tinacms/core@0.4.4
├─ tinacms@0.19.2
│  ├─ @tinacms/alerts@^0.1.3
│  ├─ @tinacms/core@^0.8.0
│  ├─ @tinacms/fields@^0.10.2
│  ├─ @tinacms/form-builder@^0.3.3
│  ├─ @tinacms/forms@^0.4.2
│  ├─ @tinacms/icons@^0.10.0
│  ├─ @tinacms/media@^0.1.0
│  ├─ @tinacms/react-alerts@^0.1.4
│  ├─ @tinacms/react-core@^0.2.12
│  ├─ @tinacms/react-forms@^0.1.4
│  ├─ @tinacms/react-modals@^0.1.4
│  ├─ @tinacms/react-screens@^0.1.3
│  ├─ @tinacms/react-sidebar@^0.1.4
│  ├─ @tinacms/react-toolbar@^0.1.2
│  ├─ @tinacms/styles@^0.4.2
│  └─ react-tinacms-editor@^0.5.2

```

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser |     Firefox    |
| os      |     Windows 10    |
| node    |   13.9.0      |
| npm     |     6.13.7    |

**Comment by ncphillips** [comment]
Thank @amykapernick ! There’s a PR in the work to fix this. It will be hotreleased in a few hours

**Comment by ncphillips** [comment]
A hot-fix was put out that should fix this. Could you confirm that @amykapernick, @bshackelford?

**Comment by samullman** [comment]
Is there a way to test for this?

package.json
```
"gatsby-tinacms-remark": "^0.7.17",
```
<img width="313" alt="Screen Shot 2020-05-12 at 8 22 57 AM" src="https://user-images.githubusercontent.com/10147333/81712723-ccdfa400-9429-11ea-87fd-7388f65575b3.png">

**Comment by bshackelford** [comment]
@ncphillips Yeah, the error still exists in `v0.7.17`. 😕

**Comment by ncphillips** [comment]
Ugh. Okay, I'm going to try converting that file to a js file. I don't quite understand how Gatsby goes about finding those

**Comment by ncphillips** [comment]
Got #1130 open 🤞

**Comment by ncphillips** [comment]
Merged and published. I'm cautiously optimistic. Sorry everyone 😞

**Comment by bshackelford** [comment]
@ncphillips That worked! Thanks for the quick fix. Sorry for the disruption. 😅

**Comment by ncphillips** [comment]
@all-contributors please add @amykapernick for bugs
@all-contributors please add @bshackelford for bugs
@all-contributors please add @samullman for bugs

**Comment by ncphillips** [comment]
Darn :P 

@all-contributors please add @amykapernick for bugs

**Comment by ncphillips** [comment]
Thanks everyone! 

Sorry for the inconvenience and the notifications

**Comment by kendallstrautman** [comment]
Fixed via #1130. Will be available in next weeks release?

**Comment by bshackelford** [comment]
@kendallstrautman Confirming this is fixed in `v0.7.18` 👍
